nx/lib/nx.ex

```diff
@@ -3265,6 +3265,129 @@ defmodule Nx do
     end)
   end
 
+  @doc ~S"""
+  Split a tensor into train and test subsets.
+
+  `split` split must be an integer greater than zero and less than the length of the tensor.
```

Comment:
I think we got some communication noise haha
We want split to be either an integer, in which case it would behave as you have it right now; or a float, which would have the previous behavior.

Reply:
ah ok no problem. I will fix it.